source/client/RunManagement.py

```diff
@@ -0,0 +1,250 @@
+from common.Card import Card
+
+"""This file contains methods used in processing runs for Liverpool.
+
+Some Liverpool specific stuff:
+it assumes that Aces can be Hi or Low but not both (K,A,2 not allowed).
+Enforces rule that you must have two natural cards between wilds in a run.
+
+In order to preserve backwards compatibility with June 2020 distribution, it assumes that card.tempnumber is
+not passed back and forth between the client and server. This only causes ambiguity in wilds and 
+Aces at the ends of runs.   Method restoreRunAssignment takes care of this by assigning
+wilds and Aces on ends of runs the appropriate tempnumber before processRuns is called.
+To preserve info on whether Ace is assigned hi or low, if Ace is assigned low, then tempnumber is set to -1.
```

Comment:
I would think that an unassigned ace would have None as its tempnumber, so you could use a simple 1 for the low assigned ace

Reply:
I sort by tempnumber, and in wrapCards I set it to a value when card is initialized.  Using value = -1 is  easier.